src/status_im/contexts/wallet/home/style.cljs

```diff
@@ -8,12 +8,11 @@
    :padding-bottom     12})
 
 (def overview-container
-  {:height 86})
+  {:height 82})
 
 (def accounts-list
-  {:padding-top    32
-   :padding-bottom 12
-   :max-height     112})
+  {:padding-top 8
+   :max-height  92})
```

Comment:
I'm curious about these height/padding changes. Is it from the designs?

Reply:
I checked everything with figma 1-1 and it was sitting right like this from what I could see.

Reply:
actually, I think I will double check this again. If anything we should not be setting max-height like this and using margins for the spacing instead. The height of the cards changes depending on the content used. Same for many of the components. Unfortunately for a lot of these quo components we were using height as margin but this is not desirable. Will revisit

Reply:
fixed this @OmarBasem - I instead use margin/padding to handle the spacing there so it is dynamic now 👍

Reply:
Thanks for checking and fixing this @J-Son89

Btw, I think the default value of `flex-grow` is 0

Reply:
I think in most cases yes but here I think the wrapper component is doing something as I needed to add this.